documentation/content/developmentworkflow.rst

```diff
@@ -120,3 +120,74 @@ To configure this Git pre-commit hook:
 
 The next time you try to commit your work, your pre-commit hook will run both
 shellcheck and shfmt, and warn you of any errors you may have made.
+
+Kwreview
+========
+Another way to have your code checked against ``shfmt`` and ``shellcheck`` is to
+use ``scripts/kwreview.sh``. By default, it will check your current patch (i.e.
+what has changed since the branch unstable) and print ``shellcheck``'s warnings
+and ``shfmt``'s warnings as diffs. Use ``kwreview -w`` to apply ``shfmt``'s
+changes to the files. Make sure you have ``shfmt``, ``shellcheck`` and
+``reviewdog`` installed (see :ref:`dependencies_label`).
+
+For vim users, it is possible to use ``kwreview.sh`` to populate the
+`quickfix <http://vimdoc.sourceforge.net/htmldoc/quickfix.html>`_ (or locations)
+window, making it easy to navigate the warnings and errors. As an example,
+consider adding the following lines to your ``.vimrc``::
+
+  function Kwreview(...)
+    let arg = get(a:, 1, "")
+    let &l:makeprg="scripts/kwreview.sh " . arg
+    make
+    execute ":e"
+  endfunction
+
+  command -nargs=* Kwreview :call Kwreview(<q-args>)
+
+Now you can call ``kwreview.sh`` with the vim command ``:Kwreview``
+(notice the upper case initial) and navigate the errors/warnings with
+``:copen``, ``:cn``, ``:cp``, etc.
+
+.. _dependencies_label:
+
+Dependencies
+============
+
+Installing ShellCheck
+~~~~~~~~~~~~~~~~~~~~~
+Shellcheck is available as a package for most distributions. On debian based
+distros, use ``sudo apt install shellcheck``. On Arch based distros, use ``sudo
+pacman -S shellcheck``.
+
+Installing shfmt
+~~~~~~~~~~~~~~~~
+
+To install shfmt, ``go`` must be installed. On debian based distros::
+
+  sudo apt install golang
+
+On arch based distros::
+
+  sudo pacman -S go
+
+With go installed, run the following command to install shfmt::
+
+  GO111MODULE=on go get mvdan.cc/sh/v3/cmd/shfmt
+
+More information can be found in their `github page <https://github.com/mvdan/sh>`__.
+
+Installing pre-commit
+~~~~~~~~~~~~~~~~~~~~~
+``pre-commit`` is available as a package for most distributions. On debian based
+distros, use ``sudo apt install pre-commit``. On Arch based distros, use ``sudo
```

Comment:
It looks like that we don't have this pre-commit on Ubuntu yet :(
Do you know how I can work around this?

Reply:
@rodrigosiqueira `pip install pre-commit`